components/brave_rewards/core/contribution/contribution_sku.cc

```diff
@@ -3,6 +3,7 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at https://mozilla.org/MPL/2.0/. */
 
+#include <memory>
```

Comment:
Do we need this?

Reply:
It is because unfortunately brave rewards is making use of `shared_ptr`, which happens to be a banned feature curiously.

Reply:
Yeah, legacy reasons... that's because of `std::function<>` (of which we still have some, despite https://github.com/brave/brave-browser/issues/18560) — basically, anything you pass to it must be copyable.